packages/accounts/scripts/plugingen.ts

```diff
@@ -0,0 +1,151 @@
+import { type Plugin } from "@wagmi/cli";
+import { pascalCase } from "change-case";
+import dedent from "dedent";
+import {
+  createPublicClient,
+  getAbiItem,
+  getContract,
+  http,
+  type AbiItem,
+  type Chain,
+} from "viem";
+import { IPluginAbi } from "../src/msca/abis/IPlugin.js";
+
+type RequiredBy<TType, TKeys extends keyof TType> = Required<
+  Pick<TType, TKeys>
+> &
+  Omit<TType, TKeys>;
```

Comment:
seems useful, wanna put it in utils somewhere?

Reply:
pushed the change